lib/config-migrator.js

```diff
@@ -0,0 +1,59 @@
+/** @babel */
+
+import _ from 'lodash'
+import { heredoc } from './werkzeug'
+
+export default async function checkConfigAndMigrate () {
```

Comment:
I think that makes sense. It will have to with `await` since the builders are async.